src/keys/keypress.rs

```diff
@@ -0,0 +1,85 @@
+//! Module contianing the KeyPress struct.
+//! KeyPress is used to index keybindings.
+
+use rustwlc::KeyMod;
+use rustwlc::xkb::{Keysym, NameFlags};
+use std::hash::{Hash, Hasher, SipHasher};
+
+use std::fmt::Result as FmtResult;
+use std::fmt::{Display, Formatter};
+
+/// Structure containing keys which are pressed
+/// to trigger a keybinding.
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Can't derive `Copy` because `KeySym` doesn't have it. I'll switch us off of `KeySym` later.